lib/blue_heron/hci/commands/controller_and_baseband/write_scan_enable.ex

```diff
@@ -33,12 +33,12 @@ defmodule BlueHeron.HCI.Command.ControllerAndBaseband.WriteScanEnable do
 
   defimpl BlueHeron.HCI.Serializable do
     def serialize(%{opcode: opcode, scan_enable: scan_enable}) do
-      <<opcode::binary, scan_enable::little-16>>
+      <<opcode::binary, 1, scan_enable>>
```

Comment:
```suggestion
      <<opcode::binary, scan_enable>>
```

Looking at the spec, it seems `scan_enable` needs to be 1 octet (byte). I couldn't find anything about the `1` byte thats being put in, so I think the fix here is to just downsize the `scan_enable` parameter from 16 bits to 8 bits

Reply:
Yep, I missed it at first as well 😄  But the `1` byte is the `Parameter Total Length` field, as described in section 5.4.1 (v5.2 of Bluetooth spec)

Reply:
oooo! duh 🤦 

Guess it's been too long since I was deep into BLE spec. Thanks @trarbr